src/icp_canister_backend/tests/test.rs

```diff
@@ -115,15 +116,15 @@ fn test_deposit_flow() {
     let user = Principal::from_text("xkbqi-2qaaa-aaaah-qbpqq-cai").unwrap();
     let deposit_amount = Nat::from(100_000_000u64);
 
-    let current_episode = get_current_episode(&pic, canister_id, user);
+    let stakable_episode = get_stakable_episode(&pic, canister_id, user, 0);
```

Comment:
There is no reason to rename current_episode to stakable, as it is still a current episode in a sense..